sema/subyt/sources.py

```
@@ -184,9 +194,9 @@
 
         return IterProxy(self)
 
-    def __exit__(self):
+    def __exit__(self, *exc):
```

Ruff (0.8.2)

Missing return type annotation for special method `__exit__`

(ANN204)